### ISSUE TITLE
BORIS wont open at all after installation

### ISSUE THREAD

**Issue by mbed9593** [olivierfriard/BORIS#526]
I am working with Windows 10 Pro operating system
On a Dell Latitude 5400
RAM 32 GB
And trying to get BORIS v.7.13.8

The boris_error.log attached
[boris_error.log](https://github.com/olivierfriard/BORIS/files/9411303/boris_error.log)

Thanks in advance for any help you can offer

**Comment by olivierfriard** [comment]
Did you try with the setup or portable version?

Can you try the v. 8.7 ?

**Comment by mbed9593** [comment]
Hi #oliverfriard,
I tried the setup version and then the portable version, and I got the same issue.
I now tried the v.8.7 and I still get the issue. See the new error log attached
[boris.log](https://github.com/olivierfriard/BORIS/files/9501378/boris.log)

**Comment by olivierfriard** [comment]
Did you get BORIS running on your system?

**Comment by mbed9593** [comment]
No, I had the University IT check the computer and I was still unable to run BORIS on the system
